app/jobs/fetch_resources_job.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# Load JSON resources from a url, persist them, and index them
+class FetchResourcesJob < ApplicationJob
+  queue_as :default
+
+  def perform(url, exhibit)
+    resp = Faraday.get(url)
+
+    resource = resp.body.split("\n")
+    resource.each_with_index do |item, index|
+      create_or_update_resource(item, exhibit, index)
+    end
+    logger.info("#{resource.count} records were created from #{url}.")
+  end
+
+  private
+
+  def create_or_update_resource(item, exhibit, index)
+    json = JSON.parse(item)
+    resource = DlmeJson.find_or_initialize_by(url: json['id'], exhibit: exhibit)
+    resource.data = { json: item }
+
+    raise "Resource #{index + 1} in #{url} is not valid." unless resource.valid?
```

Comment:
The user of the website is never going to see this error.   Do we need to write some persistent notification the database (could be a new ticket)?

Reply:
I figured it would be Honeybadger. Since all data should already be validated IR, I would expect failures to be bugs.